.github/workflows/publish.yaml

```diff
@@ -1,14 +1,14 @@
 # Single workflow to increment Ersilia version in pyproject.toml,
 # tag and push the commit to the repository, release to GitHub, PyPI, Conda, and DockerHub
 # This job updates _static_version.py, pyproject.toml, CITATION.cff, and codemeta.json
+# it update the realeasedate in the CITATION.cff
```

Comment:
I think it's fine to not add this line. We can remove this.

Reply:
This is well noted and the comment has been deleted.
@DhanshreeA